paperqa/readers.py

```diff
@@ -48,7 +48,7 @@ def chunk_pdf(
 
     if not parsed_text.content:
         raise ImpossibleParsingError(
-            "No text was parsed from the document: either empty or corrupted."
+            f"No text was parsed from the document {doc}, either empty or corrupted."
```

Comment:
Dup?

Reply:
Oh wow nice catch. You are on a roll tonight. I will revert this change

Reply:
Man actually bad rebase, this like is a once in a 3 month mistake I make. Nice catch here

Reply:
Alright, resolved this